spec/services/google_service/client_service_spec.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe GoogleService::ClientService, type: :service do
+  context 'when querying a simple keyword' do
+    it 'returns an HTTParty Response' do
```

Comment:
Can we add the cassete in the test itself
```
it 'returns an HTTParty Response', vcr: 'google_search' do
end
```

Reply:
Much clearer tests! Thanks!